tests/schemas/test_unit_schemas.py

```diff
@@ -412,6 +412,26 @@ def test_user_create():
             password="asd",
             slurm_accounts=["foo", "bar", "foo", "rab"],
         )
+    with pytest.raises(ValidationError):
+        # empty string
+        UserCreate(
+            email="a@b.c",
+            password="asd",
+            slurm_accounts=["foo", "     ", "bar"],
+        )
+    user = UserCreate(
+        email="a@b.c",
+        password="asd",
+        slurm_accounts=["f o o", "  bar "],
+    )
+    assert user.slurm_accounts == ["f o o", "bar"]
```

Comment:
I am quite sure this string (`f o o`) would not work as a SLURM account, but let's not fix any single tiny detail here.

In the future we may opt for a more strict validation of string payloads, since spaces are often not accepted anywhere in the string (leading/trailing or in the middle).